### PR TITLE
replace blockdiag with graphviz in docs

### LINKED ISSUE #9134: Replace blockdiag
In docs/tracing_reference.rst we use blockdiag to create some diagrams. The project is unmaintained now and causing issues upgrading to latest versions of sphinx.

We need someone to replace these diagrams with an alternative. In the Sphinx documentation, they recommend GraphViz: https://chiplicity.readthedocs.io/en/latest/Using_Sphinx/UsingGraphicsAndDiagramsInSphinx.html

So, the blockdiag statements need to be replaced with graphviz ones.

### PR DESCRIPTION
**Pull request by Cycloctane**
## What do these changes do?

- replace blockdiag with graphviz in docs/tracing_reference.rst
- remove `sphinxcontrib-blockdiag` dependency in requirements

## Related issue number

Fixes #9134 

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names.
- [x] Add a new news fragment into the `CHANGES/` folder
  * name it `<issue_or_pr_num>.<type>.rst` (e.g. `588.bugfix.rst`)
  * if you don't have an issue number, change it to the pull request
    number after creating the PR
    * `.bugfix`: A bug fix for something the maintainers deemed an
      improper undesired behavior that got corrected to match
      pre-agreed expectations.
    * `.feature`: A new behavior, public APIs. That sort of stuff.
    * `.deprecation`: A declaration of future API removals and breaking
      changes in behavior.
    * `.breaking`: When something public is removed in a breaking way.
      Could be deprecated in an earlier release.
    * `.doc`: Notable updates to the documentation structure or build
      process.
    * `.packaging`: Notes for downstreams about unobvious side effects
      and tooling. Changes in the test invocation considerations and
      runtime assumptions.
    * `.contrib`: Stuff that affects the contributor experience. e.g.
      Running tests, building the docs, setting up the development
      environment.
    * `.misc`: Changes that are hard to assign to any of the above
      categories.
  * Make sure to use full sentences with correct case and punctuation,
    for example:
    ```rst
    Fixed issue with non-ascii contents in doctest text files
    -- by :user:`contributor-gh-handle`.
    ```

    Use the past tense or the present tense a non-imperative mood,
    referring to what's changed compared to the last released version
    of this project.

### CODE REVIEW

**Comment by Cycloctane** [comment]
Note that GraphViz requires [additonal packages](https://graphviz.org/download/) to be installed to render diagrams when building documentation. This change haven't been presented yet.

File: docs/conf.py [hunk 1]
```diff
@@ -56,8 +56,8 @@
     "sphinx.ext.extlinks",
     "sphinx.ext.intersphinx",
     "sphinx.ext.viewcode",
+    "sphinx.ext.graphviz",
```
Comment (Dreamsorcerer) [inline docs/conf.py]: Let's keep it alphabetical.
```suggestion
    "sphinx.ext.graphviz",
    "sphinx.ext.intersphinx",
    "sphinx.ext.viewcode",
```

File: requirements/broken-projects.in
Comment (Dreamsorcerer) [inline requirements/broken-projects.in:1]: Let's delete the file and references to it.

**Review by Dreamsorcerer — CHANGES_REQUESTED** [review]
If you follow the link from the RTD job in the PR (https://aiohttp--9359.org.readthedocs.build/en/9359/tracing_reference.html) you can preview the docs. Currently, there are no diagrams rendered.

**Comment by Dreamsorcerer** [comment]
> If you follow the link from the RTD job in the PR (https://aiohttp--9359.org.readthedocs.build/en/9359/tracing_reference.html) you can preview the docs. Currently, there are no diagrams rendered.

I'm guessing that might because of your comment about needing another package, though I'm a little surprised the build doesn't fail if a dependency is missing...

File: CHANGES/9359.contrib.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Changed diagram images generator from blockdiag to GraphViz.
```
Comment (Dreamsorcerer) [inline CHANGES/9359.contrib.rst]:
```suggestion
Changed diagram images generator from ``blockdiag`` to ``GraphViz``.
```

**Comment by Dreamsorcerer** [comment]
I think that works well enough. Thanks for that.

**Comment by Dreamsorcerer** [comment]
Could you handle the backport as well?

**Pull request merged**